iron-query-params.html

```diff
@@ -69,7 +69,12 @@
     },
     _decodeParams: function(paramString) {
       var params = {};
-      var paramList = (paramString || '').split('&');
+      
+      // Work around a bug in decodeURIComponent where + is not
+      // converted to spaces:
+      paramString = (paramString || '').replace(/\+/g, '%20');
```

Comment:
No, the encoding should be left as is (encoding to %20 is more common, and should be preferred since decodeUriComponent already knows how to handle that.

All implementations that I am aware of that support + also support %20.